src/components/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -19,8 +19,10 @@ const ConfirmationDialog: React.FC<Props> = ({ open, title, body, onConfirm, onC
   const { t } = useTranslation('common');
 
   return (
-    <Dialog open={open} onClose={onClose}>
-      <h2 className={styles.title}>{title}</h2>
+    <Dialog open={open} onClose={onClose} role="dialog" aria-labbeledby="confirmation-heading">
```

Comment:
Is [alertdialog](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alertdialog_role) a better role for the confirmation dialog?
```suggestion
    <Dialog open={open} onClose={onClose} role="alertdialog" aria-labbeledby="confirmation-heading">
```

Reply:
Ah I now do agree, since we do want to demand the user's immediate attention. Thanks!